pmd-core/src/main/java/net/sourceforge/pmd/renderers/XMLRenderer.java

```diff
@@ -152,6 +152,17 @@ public void end() throws IOException {
                 writer.write(buf.toString());
             }
         }
+        
+        // config errors
+        for (final Report.ConfigurationError ce : configErrors) {
+            buf.setLength(0);
+            buf.append("<configerror ").append("rule=\"");
```

Comment:
yes, tools such as jenkins are possibly going to break, but that's why this change is ok for 6.0.0.

We should probably create an XML Schema for this report and expose it...